integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/context/CcInstance.java

```diff
@@ -179,11 +186,11 @@ public Builder withClientCertValidation() {
             return this;
         }
 
-        public CcInstance build() throws IOException, CCTestException {
+        public CcInstance build(boolean isInitialStartUp) throws IOException, CCTestException {
```

Comment:
Since we do the builder pattern here we do not pass parameter here. Needs to introduce a separate method like markInitialStartup and move on. (Then you wont have to change where setting it to false)